cli/src/main/java/org/aya/cli/library/LibraryCompiler.java

```diff
@@ -47,116 +75,222 @@
     return buildRoot.resolve("deps").resolve(depName + "_" + version);
   }
 
-  private void make(@NotNull LibraryConfig config) throws IOException {
-    // TODO[kiva]: move to package manager
-    var compiledModulePath = DynamicSeq.<Path>create();
-    var modulePath = DynamicSeq.<Path>create();
-    for (var dep : config.deps()) {
+  private @NotNull LibrarySource resolveImports(
+    @NotNull LibraryCompiler owner,
+    @NotNull Path path
+  ) throws IOException {
+    var program = AyaParsing.program(owner.locator, owner.reporter, path);
+    var source = new LibrarySource(owner, path);
+    var finder = new ImportResolver(owner, source);
+    finder.resolveStmt(program);
+    return source;
+  }
+
+  private @NotNull MutableGraph<LibrarySource> resolveLibraryImports() throws IOException {
+    var graph = MutableGraph.<LibrarySource>create();
+    var sources = collectSource(library.librarySrcRoot());
+    reporter.reportString("  [Info] Resolving source file dependency");
```

Comment:
Why this indentation?

Reply:
I also see other `reportString`s indented, but I wonder how the overall output would look like.

Reply:
![image](https://user-images.githubusercontent.com/9902643/142798180-095b580c-a023-4615-a904-bc2584968fde.png)

Reply:
I see. Could these outputs possibly nest? If so, probably you wanna use `Doc`